src/python/crash_analysis/stack_parsing/stack_analyzer.py

```diff
@@ -1962,11 +1963,19 @@ def get_crash_data(crash_data, symbolize_flag=True):
       continue
 
   # Only get repo.prop if we have an Android kernel or KASAN crash
-  if not state.android_kernel_repo and (found_android_kernel_crash or is_kasan):
-    repo_prop_path = android_settings.get_repo_prop_path()
-    if repo_prop_path and os.path.exists(repo_prop_path):
-      state.android_kernel_repo = utils.read_data_from_file(
-          repo_prop_path, eval_data=False).decode()
+  if not (state.android_kernel_prefix and state.android_kernel_hash) and (
```

Comment:
No need to use android_kernel_prefix and android_kernel_hash in state class. this is only done at end, so it can be just like

if found_android_kernel_crash or is_kasan:
  android_kernel_prefix, android_kernel_hash = android_settings.get_kernel_prefix_and_full_hash()
  temp_crash_stacktrace = ''
    for line in state.crash_stacktrace.splitlines():
      temp_crash_stacktrace += android_kernel_utils.get_stack_frame(
          line, state.android_kernel_prefix, state.android_kernel_hash) + '\n'
    state.crash_stacktrace = temp_crash_stacktrace